sano2019/rent-a-pet
# Post new pet

what I did:
1. login, logout instead of home button
2. if you are logged in, the banner is not there
3. you can create a pet (the link is in the menu under avatar picture)
4. you can see your pets http://localhost:3000/pets/my_pets
5. can delete just the pets you created
how to test:
"1" and '2' log in with the user woland..(see seeds file) and logout
'3' create pet link (avatar picture)
'4' http://localhost:3000/pets/my_pets 
'5' create another user, create several pets, go on avatar picture at "pets", you will see all the available pets(from both of the users). f you click on each pet, you have to see the small delete recycle bean icon, just for the pets belonging to the logged user :-)
